qutip/cy/mcsolve.pyx

```diff
@@ -1,39 +1,3 @@
-#cython: language_level=3
-## cython: profile=True
-## cython: linetrace=True
```

Comment:
I'm guessing both of these profile and linetrace lines should be removed and will perhaps make mcsolve slightly faster?

Reply:
If they'd been enabled, you'd _know_ about it - linetrace makes inner-loop nogil code sloooooooow!